.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - run:
           name: Ensure we have a clean docker environment
           command: |
-            docker images "fundocker/openshift*"
+            [ -z $(docker images -q "fundocker/openshift*") ]
```

Comment:
Instead of the `[]` syntax you can use the `test` command that is more readable in this context:

```bash
test -z "$(docker images -q 'fundocker/openshift*')"
```